2019-11-09 23:37:32,724	INFO	main1	<module>	Line:11	app started
2019-11-09 23:37:32,732	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-11-09 23:37:32,755	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-11-09 23:37:32,765	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-11-09 23:38:00,689	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:38:00] "GET /ProblemSet/2de84179-ecbf-491d-b924-da4f2053c1fe/ww HTTP/1.1" 200 -
2019-11-09 23:38:01,779	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:38:01] "GET /ProblemSet/init/2de84179-ecbf-491d-b924-da4f2053c1fe/ww HTTP/1.1" 200 -
2019-11-09 23:38:01,808	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:38:01] "GET /ProblemSet/answer/2de84179-ecbf-491d-b924-da4f2053c1fe/6 HTTP/1.1" 200 -
2019-11-09 23:38:32,853	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:38:32] "GET /ProblemSet/addscore/2de84179-ecbf-491d-b924-da4f2053c1fe/106/0/1/fe0e8c652f8bfa912aed532b604f15e2 HTTP/1.1" 200 -
2019-11-09 23:38:32,868	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:38:32] "GET /ProblemSet/answer/2de84179-ecbf-491d-b924-da4f2053c1fe/4 HTTP/1.1" 200 -
2019-11-09 23:38:33,298	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
